coruscant/src/components/user-settings-tabs/InvitesSettings.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { RaisedButton } from 'material-ui'
+
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table'
+
+import Theme from '../../../style/theme'
+
+const styles = {
+  container: {
+    padding: Theme.padding.tiny
+  },
+  tableHeader: {
+    background: Theme.colors.neutlight,
+  },
+  tableHeaderColumn: {
+    fontWeight: 700
+  },
+  rowClosed: {
+    color: Theme.colors.neutlight_d2
+  },
+  acceptButton: {
+    marginRight: Theme.padding.small
+  }
+}
+
+const tableData = [
+  {
+    id: 'abc1',
+    projectName: 'drumitdown',
+    expirationDate: '2018-03-01',
+    status: 'Open',
+  },
+  {
+    id: 'abc2',
+    projectName: 'tmnt',
+    expirationDate: '2018-01-23',
+    status: 'Expired',
+  },
+  {
+    id: 'abc3',
+    projectName: 'hammer-io',
+    expirationDate: '2018-02-18',
+    status: 'Accepted',
+  }
+]
+
+
+function getTableRow(data) {
+  function acceptInvite() {
+    console.log(`TODO: Accepting invite ${data.id}...`)
+  }
+
+  function declineInvite() {
+    console.log(`TODO: Declining invite ${data.id}...`)
+  }
+  const statusOpen = (data.status.toLowerCase() === 'open')
+  const rowStyles = (statusOpen)
```

Comment:
Not sure why you are wrapping everything in parenthesis - (statusOpen), (data.status.lowerCase....)

Reply:
Personal preference. I like wrapping any conditional in parenthesis, because it's clearer it's a conditional. E.g. `condition = (left == right);` is clearer than `condition = left == right`.

Reply:
👍